lib/ndr_error/version.rb

```diff
@@ -1,4 +1,5 @@
+# frozen_string_literal: true
 # Contains the version of NdrError. Sourced by the gemspec.
 module NdrError
-  VERSION = '1.0.0'
+  VERSION = '1.0.0'.freeze
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.